Gemfile.lock

```diff
@@ -1,7 +1,12 @@
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```

Comment:
What's the reason to switch on http?

Reply:
No ideas about that. This change comes from the command `make run`.

Reply:
Got it. It's because of the Gemfile.